packages/common/src/styles/theme.ts

```diff
@@ -41,5 +48,6 @@ const themeOptions = {
 };
 
 const theme = createTheme(themeOptions);
-
+theme.shadows[1] =
```

Comment:
slightly iffy maybe?
all of the material shadows are much darker than the designs we have